src/components/RequestGroupComposition/Configuration.vue

```diff
@@ -929,9 +941,14 @@ export default {
       ) {
         // The returned instrument configs do not include any field indicating that they belong to a dither sequence, other than the offset_ra
         // and offset_dec fields in the extra params so this information might need to be added if it is included in the API response in the future.
```

Comment:
I guess this comment should be removed if we have ended up adding all the expansion params to the extra params in the response...